core-tests/src/test/java/org/mule/runtime/core/MuleEventTestCase.java

```diff
@@ -39,7 +43,10 @@
 
   @Test
   public void testEventSerialization() throws Exception {
-    Event event = getTestEvent("payload");
+    FlowConstruct flowConstruct = MuleTestUtils.getTestFlow(muleContext);
+    Event event = Event.builder(DefaultEventContext.create(flowConstruct, TEST_CONNECTOR))
```

Comment:
Can we push the creation of DefaultEventContext up to AbstractMuleContextTestCase and make it reusable?